ideascube/serveradmin/models.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.db import models
+
+from ideascube.models import JSONField
+
+
+class Setting(models.Model):
+    class Meta:
+        ordering = ['-date']
+
+    namespace = models.CharField(max_length=40)
+    key = models.CharField(max_length=40)
+    value = JSONField()
+    actor = models.ForeignKey(settings.AUTH_USER_MODEL)
+    date = models.DateTimeField(auto_now=True)
+
+    def as_boolean(self):
```

Comment:
What about adding a `default` to those methods signature?

    # We'd use the default is the value is not on the DB
    # or if casting to boolean fails.
    setting.get_as_boolean(default=False)